recipe/meta.yaml

```diff
@@ -35,21 +37,18 @@ requirements:
     - cuda-version {{ cuda_compiler_version }}
   run:
     - {{ pin_compatible('custatevec', max_pin='x') }}
```

Comment:
Looks like `custatevec` already has `run_exports`. So think we should drop this from `run`

```suggestion
```